docs/source/docs_guide.md

```diff
@@ -0,0 +1,212 @@
+# Making a documentation change
+
+**Audience**: Anyone who would like to contribute to the Fabric documentation.
+
+This short guide describes how the Fabric documentation is structured, built and
+published, as well as a few conventions one should be aware of before making
+changes to the Fabric documentation.
+
+In this topic, we're going to cover:
+* [An introduction to the documentation](#introduction)
+* [Repository structure](#repository-structure)
+* [Testing your changes](#testing-your-changes)
+* [Building the documentation locally](#building-locally)
+* [Building the documentation on GitHub and ReadTheDocs](#building-on-github)
+* [Making a change to Commands Reference](#commands-reference-updates)
+* [Adding a new CLI command](#adding-a-new-cli-command)
+
+## Introduction
+
+The Fabric documentation is written in a combination of
+[Markdown](https://www.markdownguide.org/) and
+[reStructuredText](http://docutils.sourceforge.net/rst.html), and as a new
+author you can use either or both.  We recommend you use Markdown as an easy and
+powerful way to get started; though if you have a background in Python, you may
+prefer to use RST.
+
+As part of the build process, the documentation source files are converted to
+HTML using [Sphinx](http://www.sphinx-doc.org/en/stable/), and published
+[here](http://hyperledger-fabric.readthedocs.io). There is a GitHub hook for the
+main [Fabric repository](https://github.com/hyperledger/fabric) such that any
+new or changed content in `docs/source` will trigger a new build and subsequent
+publication of the doc.
+
+Language translations are available, and these are each held in their own
```

Comment:
It might be useful to link to the international languages page here

Reply:
Good idea -- I'll add Chinese, as that one is actually plugged in right now.